src/app/components/search/SaveList.js

```diff
@@ -184,27 +154,9 @@ const SaveList = ({
 
   const handleSave = () => {
     setSaving(true);
-    const input = {};
-    let queryToBeSaved = {};
-    // If it's the tipline inbox, channels is a default filter
-    if (objectType === 'tipline-inbox' && operation !== 'UPDATE_SPECIAL_PAGE') {
-      queryToBeSaved.channels = [CheckChannels.ANYTIPLINE];
-    }
-    // If it's the unmatched media page, unmatched media is a default filter
-    if (objectType === 'unmatched-media') {
-      queryToBeSaved = { unmatched: [1], sort: 'recent_activity', sort_type: 'DESC' };
-    }
-    queryToBeSaved = { ...queryToBeSaved, ...query };
-
-    if (operation === 'UPDATE_SPECIAL_PAGE') {
-      if (objectType === 'tipline-inbox') {
-        input.tipline_inbox_filters = JSON.stringify(query);
-      } else if (objectType === 'suggested-matches') {
-        input.suggested_matches_filters = JSON.stringify(query);
-      }
-    } else {
-      input.filters = JSON.stringify(queryToBeSaved);
-    }
+    const input = {
+      filters: JSON.stringify({ ...query }),
```

Comment:
Confirm that the default filters are included in this `query`.

Reply:
Confirmed that filters from core lists are preserved when saving new lists